src/cc/syms.h

```diff
@@ -125,8 +128,11 @@ class ProcSyms : SymbolCache {
   std::unique_ptr<ProcMountNS> mount_ns_instance_;
   bcc_symbol_option symbol_option_;
 
+  static int _add_load_sections(uint64_t v_addr, uint64_t mem_sz,
+                                uint64_t file_offset, void *payload);
   static int _add_module(const char *, uint64_t, uint64_t, bool, void *);
-  bool load_modules();
+  void load_exe();
+  void load_modules();
```

Comment:
I see load_modules return value is changed from "bool" to "void". Checking the code, we did not do anything with the return value, so presumably the change is okay.